extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw

```diff
@@ -121,4 +121,44 @@
     <value>Unable to get repository at the given root path</value>
     <comment>Error text for when the git extension is unable to obtain the repository given its root path</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="FolderStatusBranch" xml:space="preserve">
+    <value>Branch: {0}</value>

```

Comment:
Do we still need to use the {Locked="{0}"} syntax in the comments?

e.g:

`<comment>{Locked="{0}"} Display the current Git branch</comment>`

Reply:
I tried to find some documentation on this last week, and came up short. I did notice that Terminal doesn't do it.
https://github.com/microsoft/terminal/blob/17a55da0f9889aafd84df024299982e7b94f5482/src/cascadia/TerminalControl/Resources/es-ES/Resources.resw#L190

Reply:
I'm not familiar with this Locked syntax. Got a link?